### PR TITLE
Create Homework

### PR DESCRIPTION
**Pull request by Artur-Badalyan**
Homework in directory Day1 Day2

### CODE REVIEW

File: Artur_Badalyan/Day1/script [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+       DIR=${eachfile:0:4}
+   
+       if [ ! -d "$DIR" ]; then
+           mkdir ${eachfile:0:4}
+      fi
+  
+      mv $eachfile ./$DIR
+  done
+
+
```
Comment (araksya-hambaryan-bfg) [inline Artur_Badalyan/Day1/script:52]: Please separate tasks in different scripts

File: Artur_Badalyan/Day2/dateScript [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          ;;
+    "minute" )
+          echo $(date +"Year: %Y, Month: %m, Day: %d TIME: %T" -d "+$2 minute")
+          ;;
+    "second" )
+          echo $(date +"Year: %Y, Month: %m, Day: %d TIME: %T" -d "+$2 second")
+          ;;      
+esac          
+
+
```
Comment (araksya-hambaryan-bfg) [inline Artur_Badalyan/Day2/dateScript:28]: no check for input values
the default case is missing

File: Artur_Badalyan/Day2/gameScript [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    then        
+        printf "Your input number small in memorable number\n"
+    elif [ $input -gt $number ]
+    then        
+        printf "Your input number great in memorable number\n"        
+    fi
+done
+
+printf "Memorable number is $number\n Try again!!\n"
+
```
Comment (araksya-hambaryan-bfg) [inline Artur_Badalyan/Day2/gameScript:24]: - need to inform the user about the range of numbers
- need to finish the game if the number is correct (e.g. during the first attempt)
- no check for input values

**Review by araksya-hambaryan-bfg — CHANGES_REQUESTED** [review]

File: Artur_Badalyan/Day2/gameScript (hunk 3 above)
Comment (garik-iskandaryan-im) [inline Artur_Badalyan/Day2/gameScript:33]: - need to inform the user about the range of numbers
- need to finish the game if the number is correct (e.g. during the first attempt)
- no check for input values

File: Artur_Badalyan/Day2/dateScript (hunk 2 above)
Comment (garik-iskandaryan-im) [inline Artur_Badalyan/Day2/dateScript:39]: - no check for input values
- the default case is missing

File: Artur_Badalyan/Day1/script (hunk 1 above)
Comment (garik-iskandaryan-im) [inline Artur_Badalyan/Day1/script]: Please separate tasks in different scripts

**Review by garik-iskandaryan-im — CHANGES_REQUESTED** [review]

**Review by garik-iskandaryan-im — APPROVED** [review]

**Pull request merged**